WordPress/Classes/ViewRelated/Notifications/Style/WPStyleGuide+Notifications.swift

```diff
@@ -229,9 +226,6 @@ extension WPStyleGuide {
         fileprivate static let snippetParagraph         = NSMutableParagraphStyle(
             minLineHeight: snippetLineSize, maxLineHeight: snippetLineSize, lineBreakMode: .byWordWrapping, alignment: .natural
```

Comment:
I tested it and it seemed to work with multiple lines set. Does the server ever truncate snippets itself? I was testing yesterday and it looked like some titles were getting truncated after about 1.5 lines. Perhaps we wanted to avoid double `...`?